german/the-black-hack.md

```diff
@@ -204,7 +204,7 @@ Für jede Spielsitzung / jeden Dungeonlevel / jedes wichtige Ereignis können Ch
 
 Wenn Charaktere eine Stufe aufsteigen, erhöht sich das Maximum für ihre **Trefferpunkte** um einen Wert, der mit dem **Trefferwürfel** ihrer Klasse ausgewürfelt wird. Außerdem können Spieler für jede **Eigenschaft** einen W20 würfeln; wenn das Ergebnis höher ist als der entsprechende Wert, erhöht sich die **Eigenschaft** um 1.
 
-## Überladung
+## Überlastung
```

Comment:
Da habe ich keine Präferrenz in die eine oder die andere Richtung. Welche Vorteile siehst du für Änderung?